chapter05/Task_5_3.py

```diff
@@ -0,0 +1,120 @@
+# Who is your daddy?!)
+# Program about
+# Bloodline. You can find the father
+
+son_father = [('Den', 'Konstantin'),
+              ('Daniel', 'Pavel'),
+              ('Vladimir', 'Viktor'),
+              ('Konstantin', 'Feofan'),
+              ('Viktor', 'Yiriy')]
+
+
+while True:
+    choice = input('''
+            Hwo is your Daddy?!)
+        I know your father. You can enter
+        name, and i say, hwo is father.
+        
+    1. Get to know your father
+    2. Added pair Son - Father in base
+    3. Change pair Son - Father in base
+    4. Delete pai Son - Father from base
+    5. Show database\n
+    ''')
+
+    if choice == '1':
+        input_son = input("Enter son's name\n")
+        for pair in son_father:
+            son, father = pair
+            if input_son == son in pair:
+                print("Father name's {}\n".format(father))
+                choice = input("If it's the wrong pair, press 'n'\n")
+                if choice.lower() == 'n':
+                    continue
+                else:
+                    break
+            elif input_son not in pair:
+                continue
+        else:
+            print("This pair is not in the database\n")
+
+    elif choice == '2':
+        input_son = input("Enter son's name\n")
+        input_father = input("Enter his father name\n")
+        for pair in son_father:
```

Comment:
If instead you write:
`for son, father in son_father:`
You won't need an additional line to unpack
Same everywhere else.